src/sql/magic.py

```diff
@@ -678,8 +678,11 @@ def _persist_dataframe(
 
 def set_configs(ip, file_path):
     """Set user defined SqlMagic configuration settings"""
+    is_notebook = ip.__class__.__name__ == "ZMQInteractiveShell"
     sql = ip.find_cell_magic("sql").__self__
-    user_configs = util.get_user_configs(file_path, ["tool", "jupysql", "SqlMagic"])
+    user_configs = util.get_user_configs(
+        file_path, ["tool", "jupysql", "SqlMagic"], is_notebook
```

Comment:
also, do we need to pass this `["tool", "jupysql", "SqlMagic"]`? is there any situation where we call `get_user_configs` with a different value? if not, it should not be a parameter

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/823/commits/a43c46c3386ad2950b03cf4ee89fccca43357913)